packages/zoltan2/test/sphynx/Sphynx_Research_Driver.cpp

```diff
@@ -516,50 +410,43 @@ int main(int narg, char *arg[])
 
       sphynxParams->set("sphynx_verbosity", verbosity);
       sphynxParams->set("sphynx_max_iterations", max_iters);
-      if(block_size > 0){
-        sphynxParams->set("sphynx_block_size", block_size);
-      }
+      sphynxParams->set("sphynx_ortho_freq", orthoFreq);
+      sphynxParams->set("sphynx_res_freq", resFreq);
       sphynxParams->set("sphynx_skip_preprocessing", true);
-      //sphynxParams->set("sphynx_eigensolver", eigensolve); //TODO: Uncomment when randomized solver in Anasazi. 
+      sphynxParams->set("sphynx_eigensolver", eigensolve);
+      if (block_size > 0) sphynxParams->set("sphynx_block_size", block_size);
       if (ptype != "") sphynxParams->set("sphynx_problem_type", ptype);
       if (init != "") sphynxParams->set("sphynx_initial_guess", init);
       if (prec != "") sphynxParams->set("sphynx_preconditioner_type", prec);
       if (tol != -1) sphynxParams->set("sphynx_tolerance", tol);
-      
+
       using problem_type = Zoltan2::SphynxProblem<adapter_type>; //We found sphynx
       Teuchos::RCP<problem_type> problem;
       pComm->barrier();
       {
-        Teuchos::TimeMonitor t1b(*Teuchos::TimeMonitor::getNewTimer("Partitioning::All"));
+        Teuchos::TimeMonitor t(*Teuchos::TimeMonitor::getNewTimer("Partitioning::All"));
         {
-          Teuchos::TimeMonitor t2b(*Teuchos::TimeMonitor::getNewTimer("Partitioning::Problem"));
+          Teuchos::TimeMonitor t2(*Teuchos::TimeMonitor::getNewTimer("Partitioning::Problem"));
           problem = Teuchos::rcp(new problem_type(adapter.get(), params.get(), sphynxParams, Tpetra::getDefaultComm()));
         }
         {
-          Teuchos::TimeMonitor t3b(*Teuchos::TimeMonitor::getNewTimer("Partitioning::Solve"));
+          Teuchos::TimeMonitor t3(*Teuchos::TimeMonitor::getNewTimer("Partitioning::Solve"));
           if (vector_file ==""){
             if(me == 0)
               std::cout << eigensolve << "will be used to solve the partitioning problem." << std::endl;
             problem->solve();
           }
           else{
-            std::cout << "Problem to be partitioned with user-provided eigenvectors." << std::endl;
-            problem->setUserEigenvectors(V);
-            problem->solve();
```

Comment:
Does this work if we put it back?

Reply:
No, but I'm trying to figure out why

Reply:
Deleted the option to pass in user-specified eigenvectors.